rasa/shared/core/constants.py

```diff
@@ -89,12 +89,14 @@
 PREVIOUS_FLOW_SLOT = "rasa_previous_flow"
 CANCELLED_FLOW_SLOT = "rasa_cancelled_flow"
 CORRECTED_SLOTS_SLOT = "rasa_corrected_slots"
+RETURN_VALUE_SLOT = "RETURN_VAL"
```

Comment:
aah not sure we should have all caps slots - i'd prefer if we stick to lower case because I am not sure we'd handle capitalization properly everywhere (at least we don't test for it)